### DISCUSSION TITLE
Setting to turn off blocks detection in the background

### DISCUSSION THREAD

**Discussion by htmgarcia** [publishpress/publishpress-blocks#1236] [Feature Requests]
Setting to turn off advgb_blocks_list option update in the background.

**Comment by webd-uk** [comment]
Hi,

Can someone please let me know if anything is happening with this?

Our client https://hofnar.com has your plugin running and `advgb_blocks_list` option is set to autoload and is 61.16Kb. This is more than half (100Kb) that then triggers Wordpress to recommend the site uses a persistent object cache in "Dashboard - Tools - Site Health". Please advise if you have planned to save this data somewhere else so the option can be deleted or, at the very least, convert it to "autoload: false".

The former is much more preferable but please note that the option will have to be deleted and re-added as autoload cannot be changed once an option exists.

Thank you,

Oliver

**Reply by htmgarcia** [reply]
Hi @webd-uk,
please go your datatabase and manually delete `advgb_blocks_list` record. Then in your WordPress admin, edit a post to allow our plugin rebuild this option. It should save with `autoload` as `false`.

**Comment by webd-uk** [comment]
Thank you.

Oliver
